G/GMT/build_tarballs.jl

```diff
@@ -0,0 +1,56 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GMT"
+version = v"6.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/GenericMappingTools/gmt/releases/download/$(version)/gmt-$(version)-src.tar.gz", "ab7062912aeead1021770fad4756e0a99860fde8ea9b428fb00c22fa15a3bbfc")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd gmt-6.2.0/
+mkdir build
+cd build/
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DHAVE_QSORT_R_GLIBC=False -DHAVE___BUILTIN_BSWAP16=False -DHAVE___BUILTIN_BSWAP32=False -DHAVE___BUILTIN_BSWAP64=False   .. 
+make -j${nproc} 
+make install 
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "macos"; ),
+]
```

Comment:
Any hope for Windows, which I think is the more problematic platform?

Reply:
Sure, windows is usually the trouble-maker (even when [GMT.jl](https://github.com/GenericMappingTools/GMT.jl) appears easier to install on windows than on Mac/Linux). Ideally, we would thus have a Windows-build as well.

I made some progress meanwhile, by using an older version of NetCDF (4.7.1), and applying a few patches (`mingw` calls it `<windows.h>` instead of `<Windows.h>` etc.). 
Yet, I still run int a mingw error after a while. 

I'll upload these changes as soon as I have time; perhaps you or someone else have suggestions on how to fix that